lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -1076,6 +1076,35 @@
             "output": {
                 "shape": "ObjectMetadataObject"
             }
+        },
+        "GetBucketCseq": {
```

Comment:
We can later extend this route to return more (or all) available info eventually, so could be renamed (something like GetBucketMetadataInfo?)

Reply:
I think this route only gets the raft session information for a given bucket. The response contains aseq/cseq/vseq, etc. I'm also selecting only cseq on response, but if there is a need for this information elsewhere, I can expand it here for future use case 👍